lib/pages/chat_list/chat_list_body.dart

```diff
@@ -31,224 +28,200 @@ class ChatListViewBody extends StatelessWidget {
     final userSearchResult = controller.userSearchResult;
     final client = Matrix.of(context).client;
 
-    return PageTransitionSwitcher(
```

Comment:
Why did you delete it? It's better about performance

Reply:
Why it's better for performance ?. I think currently we don't use the animation, so just remove it for now